test/metabase/server/routes/index_test.clj

```diff
@@ -45,5 +45,32 @@
             "translations" {"" {"Metabase" {"msgid" "Metabase", "msgstr" ["Metabase"]}}}}
            (some->
             (binding [i18n/*user-locale* "en"]
-              (#'index/load-localization))
+              (#'index/load-localization nil))
             json/parse-string)))))
+
+(deftest override-localization-test
+  (testing "a valid override is honored no matter what the user locale is"
+    (is (= {"charset"      "utf-8"
+            "headers"      {"mime-version"              "1.0"
+                            "content-type"              "text/plain; charset=UTF-8"
+                            "content-transfer-encoding" "8bit"
+                            "x-generator"               "POEditor.com"
+                            "project-id-version"        "Metabase"
+                            "language"                  "es"
+                            "plural-forms"              "nplurals=2; plural=(n != 1);"}
+            "translations" {"" {"Your database has been added!" {"msgstr" ["¡Tu base de datos ha sido añadida!"]}}}}
```

Comment:
Probably not likely, but I think if a translator ever revises this particular translation, the test will fail, I think.